roles/add_livemigration_user/README.md

```diff
@@ -4,7 +4,7 @@ This role sets the live migration user on the cluster, with ssh key exchanges
 
 ## Requirements
 
-no requirement.
```

Comment:
Why do you need this role as a dependency ?
There is nothing in the role that uses distro specific variables

Reply:
Removed, this was forgot after rebasing on the new debiantcentos branch after merge of https://github.com/seapath/ansible/pull/643